localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/README.md

```diff
@@ -0,0 +1,61 @@
+# ndt scan matcher
+
+## Purpose
+
+ndt_scan_matcher is a package for position estimation using the NDT scan matching method.
+
+There are two main functions of this package :
+
+- estimate position by scan matching
+- estimate initial position via ros service using Monte Carlo method
+
+## Inputs / Outputs
+
+### Input
+
+| Name                       | Type                                            | Description       |
+| -------------------------- | ----------------------------------------------- | ----------------- |
+| `ekf_pose_with_covariance` | `geometry_msgs::msg::PoseWithCovarianceStamped` | initial pose      |
+| `pointcloud_map`           | `sensor_msgs::msg::PointCloud2`                 | map pointcloud    |
+| `points_raw`               | `sensor_msgs::msg::PointCloud2`                 | sensor pointcloud |
+
+### Output
+
+| Name                              | Type                                            | Description                                                                                                                              |
+| --------------------------------- | ----------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| `ndt_pose`                        | `geometry_msgs::msg::PoseStamped`               | estimated pose                                                                                                                           |
+| `ndt_pose_with_covariance`        | `geometry_msgs::msg::PoseWithCovarianceStamped` | estimated pose with covariance                                                                                                           |
+| `/diagnostics`                    | `diagnostic_msgs::msg::DiagnosticArray`         | diagnostics                                                                                                                              |
```

Comment:
```suggestion
| `diagnostics`                    | `diagnostic_msgs::msg::DiagnosticArray`         | diagnostics                                                                                                                              |
```
Question: could the slash be removed? I'm no sure.

Reply:
`/diagnostics` and `/diagnostics_agg` are the special cases.
They should be the global namespace.

Reply:
This topic is intentionally named with `/` as a global topic name.

https://github.com/tier4/AutowareArchitectureProposal.iv/blob/use-autoware-auto-msgs/localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp#L150

Reply:
FYI https://github.com/ros/diagnostics/blob/9db8532c83969d2723c83f188a130100c2b55551/diagnostic_aggregator/src/aggregator.cpp#L69-L70

Reply:
Thank you for the information! Resolve this.